{"uid":"f0e62d133aadb0fa","name":"APP-2763: Ошибка при создании борды с длинным заголовком custom field: TaskRelations","fullName":"tests.test_backend.board.board_validation_test#test_create_board_with_long_custom_field_title","historyId":"3a95a2f13d3f56e5a1a48895483f8e88","time":{"start":1749744412189,"stop":1749744412449,"duration":260},"status":"passed","statusMessage":"XPASS Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"delete_output_dir","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"owner_client","time":{"start":1749744399429,"stop":1749744400163,"duration":734},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749744400163,"stop":1749744401184,"duration":1021},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749744401184,"stop":1749744402659,"duration":1475},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_board","time":{"start":1749744405216,"stop":1749744405548,"duration":332},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","statusMessage":"XPASS Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)","steps":[{"name":"Отправка запроса с custom field title длиннее 50 символов","time":{"start":1749744412189,"stop":1749744412449,"duration":260},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"Проверка, что API вернул 400 – ошибка валидации длины заголовка","time":{"start":1749744412449,"stop":1749744412449,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":2,"attachmentsCount":0,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749744437355,"stop":1749744438003,"duration":648},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"bug","value":"APP-2763"},{"name":"tag","value":"@pytest.mark.xfail(reason='Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)')"},{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"board_validation_test"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2515-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.board_validation_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"field_type","value":"'TaskRelations'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":7,"passed":3,"unknown":0,"total":10},"items":[{"uid":"c96a2663f7070b0f","reportUrl":"https://vaizcom.github.io/autotests/124//#testresult/c96a2663f7070b0f","status":"passed","statusDetails":"XPASS Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)","time":{"start":1749744324905,"stop":1749744325111,"duration":206}},{"uid":"471169494a07a439","reportUrl":"https://vaizcom.github.io/autotests/123//#testresult/471169494a07a439","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6ed7455e50>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749744004319,"stop":1749744004319,"duration":0}},{"uid":"79fa512d8dffa306","reportUrl":"https://vaizcom.github.io/autotests/122//#testresult/79fa512d8dffa306","status":"passed","statusDetails":"XPASS Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)","time":{"start":1749743628683,"stop":1749743628873,"duration":190}},{"uid":"19335200297fbe8d","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/19335200297fbe8d","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742821432,"stop":1749742821432,"duration":0}},{"uid":"98eda1fff59bbc25","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/98eda1fff59bbc25","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742668861,"stop":1749742668861,"duration":0}},{"uid":"43f2279c80b8fe1","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/43f2279c80b8fe1","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742498676,"stop":1749742498676,"duration":0}},{"uid":"f34daf0668096fc4","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/f34daf0668096fc4","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741235041,"stop":1749741235041,"duration":0}},{"uid":"f72576a70fe6cd39","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/f72576a70fe6cd39","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740827609,"stop":1749740827609,"duration":0}},{"uid":"cf71936d2d2b3fe2","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/cf71936d2d2b3fe2","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740654302,"stop":1749740654302,"duration":0}}]},"tags":["backend","@pytest.mark.xfail(reason='Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)')"]},"source":"f0e62d133aadb0fa.json","parameterValues":["'TaskRelations'"]}